src/load.py

```diff
@@ -38,7 +49,13 @@ def lambda_handler(event, context):
     response = None
     try:
         logger.debug(event)
-        record_id = etl(event)
+        size = event['s3']['object']['size']
+        if size < os.getenv('S3_OBJECT_SIZE_LIMIT'):
```

Comment:
The plan is to remove this `if` clause in the near future right (that's what's I have in mind anyway)?

Reply:
correct